english/hit-song/hit-song.md

```diff
@@ -0,0 +1,7 @@
+You write a hit song called "No one wants to Pika-choose me".  But unfortunately your agent steals all of the royalties to your song.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.